.buildkite/rebuild.hs

```diff
@@ -233,7 +233,9 @@ setupBuildDirectory dryRun buildDir = do
 -- Remove certain files which get cached but could cause problems for subsequent
 -- builds.
 cleanBuildDirectory :: FilePath -> IO ()
-cleanBuildDirectory buildDir = findTix buildDir >>= mapM_ rm
+cleanBuildDirectory buildDir = mapM_ rm =<< do
+    findTix buildDir
+    findHie buildDir
```

Comment:
I'm a bit confused by this do-notation: this is in `IO` and isn't the results from `findTix` simply dropped?

Reply:
Err duh yes you're right - fixed with the monoid instance of `IO a`.